android/capacitor/src/main/java/com/getcapacitor/plugin/util/CapacitorHttpUrlConnection.java

```diff
@@ -275,7 +275,7 @@ private void writeFormDataRequestBody(String contentType, JSArray entries) throw
                     if (type.equals("string")) {
                         os.writeBytes(twoHyphens + boundary + lineEnd);
                         os.writeBytes("Content-Disposition: form-data; name=\"" + key + "\"" + lineEnd + lineEnd);
-                        os.writeBytes(value);
+                        os.write(value.getBytes("UTF-8"));
```

Comment:
```suggestion
                        os.write(value.getBytes(StandardCharsets.UTF_8));
```
Can you use `StandardCharsets.UTF_8` instead of hardcoding `"UTF-8"`?